### ISSUE TITLE
[arm32][5.0] llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:255:56: error: non-constant-expression cannot be na rrowed from type 'uint64_t' (aka 'unsigned long long') to 'uintptr_t' (aka 'unsigned int') in initializer list [-Wc++11- narrowing]

### ISSUE THREAD

**Issue by uraimo** [uraimo/buildSwiftOnARM#40]
```
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:255:56: error: non-constant-expression cannot be na
rrowed from type 'uint64_t' (aka 'unsigned long long') to 'uintptr_t' (aka 'unsigned int') in initializer list [-Wc++11-
narrowing]
          BuildSystemFrontendDelegate::ProcessHandle { handle.id });
                                                       ^~~~~~~~~
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:255:56: note: insert an explicit cast to silence th
is issue
          BuildSystemFrontendDelegate::ProcessHandle { handle.id });
                                                       ^~~~~~~~~
                                                       static_cast<uintptr_t>( )
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:263:56: error: non-constant-expression cannot be na
rrowed from type 'uint64_t' (aka 'unsigned long long') to 'uintptr_t' (aka 'unsigned int') in initializer list [-Wc++11-
narrowing]
          BuildSystemFrontendDelegate::ProcessHandle { handle.id },
                                                       ^~~~~~~~~
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:263:56: note: insert an explicit cast to silence th
is issue
          BuildSystemFrontendDelegate::ProcessHandle { handle.id },
                                                       ^~~~~~~~~
                                                       static_cast<uintptr_t>( )
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:272:56: error: non-constant-expression cannot be na
rrowed from type 'uint64_t' (aka 'unsigned long long') to 'uintptr_t' (aka 'unsigned int') in initializer list [-Wc++11-
narrowing]
          BuildSystemFrontendDelegate::ProcessHandle { handle.id },
                                                       ^~~~~~~~~
/mnt/buildSwiftOnARM/llbuild/lib/BuildSystem/BuildSystemFrontend.cpp:272:56: note: insert an explicit cast to silence th
is issue
          BuildSystemFrontendDelegate::ProcessHandle { handle.id },
                                                       ^~~~~~~~~
                                                       static_cast<uintptr_t>( )
```

**Comment by uraimo** [comment]
The process.id is hardcoded as uint64_t, easy fix, opening a PR later.

**Comment by uraimo** [comment]
See [PR#480](https://github.com/apple/swift-llbuild/pull/480).

**Comment by uraimo** [comment]
Fixed with [llbuild/111d9828e6422447321196b1826c0a666a0632d2](https://github.com/apple/swift-llbuild/commit/111d9828e6422447321196b1826c0a666a0632d2), not sure if it has landed yet in 5.1.

Note for self: Check if it has.

**Comment by uraimo** [comment]
Merged in 5.2, nothing else to do.
